server.go

```diff
@@ -3024,7 +3045,6 @@ func (s *server) rebroadcastHandler(ctx context.Context) {
 				time.Duration(randomUint16Number(1800)))
 
 		case <-ctx.Done():
-			close(s.quit)
```

Comment:
Why is this being removed?  It needs to be closed in order to have the desired effect when the context is done.